### ISSUE TITLE
UBC Compatibility

### ISSUE THREAD

**Issue by bookerthegeek** [oitsjustjose/Geolosys#109]
## Heyo!
Hello Again. I'm requesting some compatabliity betwen your wonderful mod and the recently updated [Underground Biomes Constructs](https://minecraft.curseforge.com/projects/underground-biomes) ([Github](https://github.com/kyeiti/UndergroundBiomesConstructs))

### Versions
- Minecraft Forge: 14.23.3.2678
- Geolosys: 1.9.1
- Underground Biomes: 1.0.1

### What happens:
Your ore generates but does not match the surrounding stone

### What should happen:
Match the surrounding stone?

### Additional Comments:
Basically similar functionality to [THIS](https://minecraft.curseforge.com/projects/ubc-ore-registrar) mod

**Comment by Dulciphi** [comment]
I'm also looking into this and would be interested to know if you've explored possibilities for compatibility yet.
 
No idea if [this](https://minecraft.curseforge.com/projects/ore-stone-variants) helps.

**Comment by oitsjustjose** [comment]
Came to update anyone here that as of 3.0.1 UBC officially supports Geolosys and vice-versa!
